src/test/kotlin/app/cash/licensee/LicenseePluginFixtureTest.kt

```diff
@@ -194,7 +194,7 @@ class LicenseePluginFixtureTest {
     val fixtureDir = File(fixturesDir, fixtureName)
     val result = createRunner(fixtureDir).buildAndFail()
     assertThat(result.output).contains(
-      "'app.cash.licensee' requires compatible language/platform plugin to be applied (project :some:thing)",
+      "Task 'licensee' not found in root project 'plugin-missing-on-subproject-fails' and its subprojects.",
```

Comment:
I really don't like this failure mode. Can we keep `afterEvaluate` for detecting this case?

Reply:
Sure, I added this check back.